### PR TITLE
New Company Added

### PR DESCRIPTION
**Pull request by gaurab-khanal**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [ ] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -544,6 +544,23 @@
         "youtube": null
       }
     },
+    {
+      "name": "Karobar App",
+      "logoName": "karobar_app.png",
```
Comment (thenishantsapkota) [inline data/companies.json:549]: The filename format is `karobarapp.png`

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/companies.json [hunk 2]
```diff
@@ -544,6 +544,23 @@
...
+      "establishedAt": null,
```
Comment (thenishantsapkota) [inline data/companies.json:550]: The establishedAt is a required field and is a unix timestamp. Please update it accordingly!

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by TheLearneer — COMMENTED** [review]
This seems more like a product rather than a company, I don't think it suits well in this repository.

For future, we might want to create another repository which lists all the products developed by nepali companies that are available to use for public.
However, for now, I don't think this entry suits well in this repository.

cc: @thenishantsapkota

**Comment by gaurab-khanal** [comment]
> This seems more like a product rather than a company, I don't think it suits well in this repository.
> 
> For future, we might want to create another repository which lists all the products developed by nepali companies that are available to use for public. However, for now, I don't think this entry suits well in this repository.
> 
> cc: @thenishantsapkota

Yeah, Just knew about this. I will edit with its Company name where it falls.

File: data/companies.json (hunk 1 above)
Comment (gaurab-khanal) [reply data/companies.json]: Just made the changes. You can review it.

**Review by TheLearneer — CHANGES_REQUESTED** [review]
The image dimension doesn't match the size mentioned in the contirubitng docs, please update it accordingly

File: data/companies.json [hunk 3]
```diff
@@ -544,6 +544,23 @@
         "youtube": null
       }
     },
+    {
+      "name": "Bytecare Technology",
```
Comment (susanhsrestha) [inline data/companies.json:548]: The information must be arranged alphabetically.

File: data/companies.json [hunk 4]
```diff
@@ -544,6 +544,23 @@
...
+      "logoName": "bytecareTechnology.png",
+      "establishedAt": 1593998205,
+      "location": "Jawalakhel, Lalitpur",
+      "website": "https://www.bytecaretech.com/",
+      "email": "contact@bytecaretech.com",
+      "phone": "+977 9815303657,9849853141",
```
Comment (susanhsrestha) [inline data/companies.json:554]: The format of the phone number is incorrect. If available, please also include a landline number.
```suggestion
      "phone": "+977-9815303657",
```

**Pull request closed without merging**